src/components/authorization/policy/conditions/aggregate.condition.ts

```diff
@@ -71,27 +74,82 @@ export class AndConditions<
 > extends AggregateConditions<TResourceStatic> {
   protected readonly iteratorKey = 'every';
   protected readonly cypherJoiner = ' AND ';
+
+  static from<T extends ResourceShape<any>>(
+    ...conditions: Array<Condition<T>>
+  ) {
+    if (conditions.length === 1) {
+      return conditions[0];
+    }
+
+    const merged = [...groupBy(conditions, byType).values()].flatMap(
+      (sames) => {
+        const same = sames[0]!;
+        return same.intersect ? same.intersect(sames) : sames;
+      },
+    );
+
+    if (merged.length === 1) {
```

Comment:
Seems to me like this should only ever be 0 or 1, so you should be able to simplify to just `if (merged.length)` right?

Reply:
It shouldn't ever be zero. We do handle no conditions better in the policy builder.
The one length is just to unwrap the aggregate
So
```
any(member, member)
```
Just becomes
```
member
```
Instead of
```
any(member)
```